packages/venia-ui/lib/components/ProductFullDetail/customAttributes.js

```diff
@@ -0,0 +1,251 @@
+                                            <RichContent
+                                                key={key}
+                                                html={option.label}
+                                            />
+                                        );
+                                    }
+                                );
+
+                                attributeContent = (
+                                    <div
+                                        className={classes.attributeContentHtml}
+                                    >
+                                        {options}
+                                    </div>
+                                );
+                            } else {
+                                const options = selected_attribute_options.attribute_option
+                                    .map(option => option.label)
+                                    .join(', ');
+
+                                attributeContent = (
+                                    <div className={classes.attributeContent}>
+                                        {options}
+                                    </div>
+                                );
+                            }
+                        } else if (entered_attribute_value?.value) {
+                            // For TEXT and TEXTAREA types
+                            if (isHtml) {
+                                // TODO: Get decoded wysiwyg widgets from GraphQl
+                                attributeContent = (
+                                    <div
+                                        className={classes.attributeContentHtml}
+                                    >
+                                        <RichContent
+                                            html={entered_attribute_value.value}
+                                        />
+                                    </div>
+                                );
+                            } else {
+                                attributeContent = (
+                                    <div className={classes.attributeContent}>
+                                        {entered_attribute_value.value}
+                                    </div>
+                                );
+                            }
+                        }
+                    }
+                }
+
+                return (
+                    <li key={key} className={classes.listItem}>
+                        <div className={classes.attributeLabel}>
+                            {attributeLabel}
+                        </div>
+
+                        {attributeContent}
+                    </li>
+                );
+            }
+        );
+    }, [
+        classes.attributeLabel,
+        classes.listItem,
+        classes.attributeContentHtml,
+        classes.attributeContent,
```

Comment:
Optional, but this identity shouldn't change, can simplify dep list.
```suggestion
        classes
```

Reply:
Done!